docs/internal/Level-1+-Reviews-and-Changes-Requested.md

```diff
@@ -0,0 +1,131 @@
+# Level 1+ Reviews and Changes Requested
+
+The [Level 1 Reviews](Level-1-Review-And-LOQ.md) explains how reviews with only 1 level works. Which is a subset of how a review with more levels (per stage) should work.
+
+The aim of this page is to outline the structure changes and flows:
+
+## Consolidation in agreement
```

Comment:
Be good if the description part of these in the list had matching tense. So either:
- Reviewer (Level 1) - reviewing an application
- Reviewer (Level 1) - submitting ~~`LOQ (List of Questions)`~~ `Conform` or `Non Conform` advise to upper level reviewer

OR:

- Reviewer (Level 1) - review an application
- Reviewer (Level 1) - submit ~~`LOQ (List of Questions)`~~ `Conform` or `Non Conform` advise to upper level reviewer

But not a mixture of both

Reply:
Ah, I see on the Leve1 review page you've used the "ing" tense in a similar list, so probably best to go with that.

Reply:
Ok. Will add `ing` to where is missing